src/main/scala/outwatch/http/Http.scala

```diff
@@ -1,10 +1,40 @@
 package outwatch.http
 
+import cats.Eval.always
 import cats.effect.IO
-import rxscalajs.Observable
-import rxscalajs.dom.{Request, Response}
+import monix.execution.Scheduler.Implicits.global
+import monix.reactive.Observable
+import org.scalajs.dom.ext.Ajax.InputData
+import org.scalajs.dom.ext.{Ajax, AjaxException}
+import org.scalajs.dom.{Blob, XMLHttpRequest}
+
+import scala.concurrent.Future
+import scala.scalajs.js
+import scala.scalajs.js.typedarray.ArrayBuffer
+import scala.scalajs.js.|
+
 
 object Http {
```

Comment:
It seems like here is more going on than a simple port. Could you explain what you did here? I'm also not using the `Http` module myself, so this should be reviewed by someone who does.

If its too different it should be a separate commit.

Reply:
`rxscala` has an `Observable.ajax` method. Monix doesn't have that, so essentially I had to re-implement that functionality (based on `org.scalajs.dom.ext.Ajax`).